### PR TITLE
Use new Link component on inventory/people and location pages

### PR DESCRIPTION
**Pull request by constanzauanini**
Screenshot 2022-12-07 at 17 13 11
Screenshot 2022-12-07 at 17 13 18
Screenshot 2022-12-07 at 17 13 25
Screenshot 2022-12-07 at 17 13 32
Screenshot 2022-12-07 at 17 13 39

### CODE REVIEW

File: src/components/common/Link.tsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ export type LinkProps = {
   onClick?: (e?: React.MouseEvent) => void;
   className?: string;
   fontSize?: string | number;
+  target?: string;
```
Comment (karthikbtf) [inline src/components/common/Link.tsx:12]: is this used?

**Review by karthikbtf — COMMENTED** [review]
Thanks Constanza!! We missed the batch details modal with link to accession id

Terraware App 2022-12-07 12-20-38

File: src/components/common/Link.tsx (hunk 1 above)
Comment (constanzauanini) [reply src/components/common/Link.tsx:12]: Oh, I forgot to commit one file

File: src/components/Inventory/view/BatchDetailsModal.tsx [hunk 2]
```diff
@@ -12,7 +12,7 @@ import { createBatch, updateBatch, updateBatchQuantities } from 'src/api/batch/b
 import { getSpecies } from 'src/api/species/species';
 import { Species } from 'src/types/Species';
 import { APP_PATHS } from 'src/constants';
-import { Link } from 'react-router-dom';
+import Link from 'src/components/common/Link';
```
[the remark this answers is not in the document]
Comment (karthikbtf) [reply src/components/Inventory/view/BatchDetailsModal.tsx:15]: ah yes, line 211, thanks!!

**Comment by constanzauanini** [comment]
> Thanks Constanza!! We missed the batch details modal with link to accession id
> 
> Terraware App 2022-12-07 12-20-38

This is the line we were talking about. It's already done:
Screenshot 2022-12-07 at 17 31 03

**Comment by hannah-cai** [comment]
the styling should actually be font-weight: 500, not 600! my bad 🥲

**Comment by karthikbtf** [comment]
> the styling should actually be font-weight: 500, not 600! my bad 🥲

i have a PR in flight, will update it.. good thing is we have just one place to update now.

**Pull request merged**